firmware/src/networking/wifi/mod.rs

```diff
@@ -0,0 +1,62 @@
+use defmt::debug;
+use embassy_futures::yield_now;
+use embedded_svc::wifi::{ClientConfiguration, Configuration, Wifi};
+use smoltcp::wire::Ipv4Address;
+
+#[cfg(feature = "esp-wifi")]
+#[path = "esp.rs"]
+pub mod ඞ;
+
+const SSID: &str = env!("SSID");
+const PASSWORD: &str = env!("PASSWORD");
+static SERVER_IP: Ipv4Address = Ipv4Address::new(192, 168, 10, 121);
```

Comment:
Should also be from env

Reply:
yeah, and how do we parse it. It's kind of a temp thing

Reply:
I think we could have a const function for that but I'm the interests of not overcomplicating, I'll resolve